### PR TITLE
Fix TagBundle

### PR DESCRIPTION
**Pull request by carltongibson**
Doctrine wraps DB specific error codes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+    * BUGFIX             #1726 [TagBundle]   Check for Doctrine Error rather than DB specific error code
```
Comment (danrot) [inline CHANGELOG.md]: Wrong alignment

**Comment by danrot** [comment]
The rest looks fine.

**Comment by carltongibson** [comment]
@danrot I changed that alignment.

**Comment by carltongibson** [comment]
Closing in favour of #1742

**Pull request closed without merging**